src/styles/blocks/star.scss

```diff
@@ -0,0 +1,98 @@
+.star {
+  display: flex;
+  flex-direction: column;
+
+  &__content {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-bottom: 12px;
+
+    &-reviews {
+      display: flex;
+      align-items: center;
+
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      margin: 0;
+    }
+  }
+
+  &__price {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin: 0 0 4px;
+
+    &-text {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $secondary-color;
+    }
+
+    &-cost {
+      display: flex;
+      align-items: center;
+
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      margin: 0;
+    }
+  }
+
+  .stars {
```

Comment:
do you need to create star and stars block? also 11'th point in checklist, but I encourage you to use one of them